DocumentFormat.OpenXml/src/Framework/OpenXmlPackage.cs

```diff
@@ -1384,6 +1384,56 @@ internal class PartUriHelper
             private Dictionary<string, int> _sequenceNumbers = new Dictionary<string, int>(20);
             private Dictionary<string, int> _reservedUri = new Dictionary<string, int>();
 
+            //Added for issue #123, some contentTypes need to have a 1 appended to the name
+            //for the first item in the package
+            private List<string> _numberedContentTypes = new List<string>
```

Comment:
For performance reasons, this should be HashSet<string> with a supplied comparer (either `StringComparer.Ordinal` or `StringComparer.OrdinalIgnoreCase` depending on if case is important). Also, make it static readonly so it's only created once for all packages